logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -165,10 +165,10 @@ function Get-DefaultConnectionStringsByEngine {
         }
         PostgreSQL = @{
             ConnectionStrings = @{
-                ((Get-ConnectionStringKeyByDatabaseTypes)[(Get-DatabaseTypes).Ods])      = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_{0};"
-                ((Get-ConnectionStringKeyByDatabaseTypes)[(Get-DatabaseTypes).Admin])    = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_Admin;"
-                ((Get-ConnectionStringKeyByDatabaseTypes)[(Get-DatabaseTypes).Security]) = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_Security;"
-                ((Get-ConnectionStringKeyByDatabaseTypes)[(Get-DatabaseTypes).Master])   = "Host=localhost; Port=5432; Username=postgres; Database=postgres;"
+                ((Get-ConnectionStringKeyByDatabaseTypes)[(Get-DatabaseTypes).Ods])      = "Host=localhost; Port=5432; Username=postgres; Database=EdFi_{0};pooling=true; Minimum Pool Size=10; Maximum Pool Size=50; client encoding=utf-8;"
```

Comment:
All four of these should have a space between the ';' and the pooling keyword to match the connection string style.  Also as per https://www.npgsql.org/doc/connection-string-parameters.html, the parameters should probably be "Pooling" and "Client Encoding".

Reply:
Note, the connection string is parsed using ignore case, culture invariant. However I made the change.

Reply:
Note pooling is defaulted as true; however i added it to the connection string to make it easier to understand it is on.